frontend/src/pages/admin/Templates/index.vue

```diff
@@ -13,7 +14,9 @@
         <ff-loading v-if="loading" message="Loading Templates..." />
         <ff-data-table v-if="!loading" :columns="columns" data-el="templates"
                        :rows="templates" :show-search="true" search-placeholder="Search Templates..."
-                       :search-fields="['name',, 'description', 'owner_username', 'owner_id']">
+                       :search-fields="['name',, 'description', 'owner_username', 'owner_id']"
```

Comment:
Is the double comma a typo here? I'm not aware of some sort of special vue syntax.

Reply:
unknown. existing code. suspect it is but it has zero affect.

Reply:
yep, legacy typo